framework/source/class/qx/bom/AnimationFrame.js

```diff
@@ -76,11 +76,13 @@ qx.Bootstrap.define("qx.bom.AnimationFrame",
      * soon as the given duration is over.
      *
      * @param duration {Number} The duration the sequence should take.
+     *
+     * @ignore(performance.*)
      */
     startSequence : function(duration) {
       this.__canceled = false;
 
-      var start = +(new Date());
+      var start = (window.performance && window.performance.now) ? (performance.now() + qx.bom.AnimationFrame.__start) : Date.now();
```

Comment:
@peuter @cajus. I'm convinced, maybe besides of the test which should be (I think)

```javascript
(window.performance && performance.now)
```
to get par with the test in the defer section.